source/content/guides/autopilot/04-troubleshoot.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Pantheon Autopilot
+subtitle: Autopilot Errors
+description: Troubleshoot Common Autopilot Errors
+categories: [automate]
+tags: [autopilot, troubleshoot, webops]
+type: guide
+layout: guide
+showtoc: true
+anchorid: troubleshoot-autopilot
+permalink: docs/guides/autopilot/troubleshoot-autopilot/
+editpath: autopilot/04-troubleshoot.md
+reviewed: "2021-07-21"
+---
+
+# Troubleshoot Autopilot Error Messages
+This page has a consolidated list of common issues and solutions you may encounter when using Autopilot.
+
+## Element Exclusions 
+Error Message: "We didn’t detect any plugins, modules or themes that were eligible for exclusion."
+ 
+### Issue
+The extensions list used for exclusions should be refreshed when the Autopilot plan is initialized for the first time or when using the slow converge feature. Since the plan is only set after Autopilot is initialized, the list is unavailable when Autopilot is started for the first time. 
+
+### Solution
+The workflow to refresh extensions can then be triggered from the GraphQL Playground with the following code change:
+
+```
+mutation {
+  refreshAvailableExtensions(
+    args: { id: "SITE_UUID"}
+  ) {
+    id
+    description
+  }
+}
+```
+
+After that workflow completes, refresh the Autopilot settings or restart the initialization wizard. The extension options should be present.
+
+
+##  Drush Version
+Error Message: "Drush version 8 required" 
+
+### Issue
+Autopilot only works on Drupal sites that are running Drush 8. Drush 5/7/9/10 are not supported.
```

Comment:
Note that this limitation does not apply to sites that are Composer-managed. Composer-managed sites should use Drush 10; they should not receive the Drush version error.

Note that currently, Autopilot only supports Integrated Composer; build tools sites cannot be updated.